.gitignore

```diff
@@ -13,4 +13,6 @@
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
-build/
```

Comment:
Why is the build directory no longer ignored? I think it should still be included if somebody builds Tosca using the command line.